templates/ts-apollo-mongodb-backend/README.md

```diff
@@ -30,15 +30,15 @@ yarn develop
 
 For more on customising your Graphback application, check out [our docs](https://graphback.dev/docs/gettingstarted)
 
-If your project contains the client application then please follow `./client/README.md` for info regarding running the client side.
+If your project contains the client application then please follow [`./client/README.md`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-mongodb-backend/README.md) for info regarding running the client side.
```

Comment:
You can hyperlink the relative path so that it won't send you to GitHub:

```suggestion
If your project contains the client application then please follow [`./client/README.md`](./client/README.md) for info regarding running the client side.
```

Reply:
Didn't know this could be done. Will take care from next time :)